contracts/true-currencies/TokenController.sol

```diff
@@ -86,6 +86,8 @@ contract TokenController {
     // pausing the contract upgrades the proxy to this implementation
     address public constant PAUSED_IMPLEMENTATION = 0x3c8984DCE8f68FCDEEEafD9E0eca3598562eD291;
 
+    address public proofOfReservePauser;
```

Comment:
The `public constant` fields above don't take up any storage slots (constants are compiled into the contract source code), so let's move this address up, just after `gasRefunder`.

Reply:
Done.